src/main/java/org/apache/cassandra/sidecar/config/impl/SSTableUploadConfigurationImpl.java

```diff
@@ -78,4 +110,20 @@ public float minimumSpacePercentageRequired()
     {
         return minimumSpacePercentageRequired;
     }
+
+    /**
+     * Returns the String representation of a set of posix file permissions used during an SSTable file upload.
+     * When an SSTable file is created the specified permissions will be used to create the file.
+     * For example, the String {@code rw-r--r--} represents the set of permissions
+     * {@link PosixFilePermission#OWNER_READ}, {@link PosixFilePermission#OWNER_WRITE},
+     * {@link PosixFilePermission#GROUP_READ}, and {@link PosixFilePermission#OTHERS_READ}.
+     *
+     * @return the String representation of a set of posix file permissions used during an SSTable file upload
+     */
```

Comment:
nit: can we simply have the following
```
    /**
     * {@inheritDoc}
     */
```

I know that the other methods in the class have the copy-pasted java docs already.. would appreciate less code, including repeated docs.

Reply:
good point, IntelliJ did the copy/paste work for me already :), but it's more work for the reviewer. I will follow your recommendation. Thanks